src/blackbox_decode.c

```diff
@@ -454,11 +454,23 @@ void outputGPSFields(flightLog_t *log, FILE *file, int64_t *frame)
     }
 }
 
+int getMajorVersion(flightLog_t *log) {
+	if (!log->private->fcVersion[0]) {
+		return -1; //Unknown version
+	}
+	char fcVersion[30];
+	strcpy(fcVersion, log->private->fcVersion);
+	return atoi(strtok(fcVersion,"."));
```

Comment:
There are still plenty of places where the formatting is not up to the coding standard. Please read https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md and format **all** of your changes accordingly.